src/mtx3/utils/CommandBinaryBuffer.ts

```diff
@@ -1101,7 +1101,71 @@ export interface IOperatorParameters {
     displaySet4: IDisplaySet4OperatorParameter
 }
 
+export interface IDefine1OperatorParameterExtended {
+    /**
+     * Reset daily maximum power using button.
```

Comment:
```suggestion
     * Reset daily maximum power.
```

or add additional description about what button and how it relates to parameter in codecs

Reply:
fixed